templates/rubocop.yml.erb

```diff
@@ -0,0 +1,256 @@
+require:
+  - rubocop-performance
+  - rubocop-rails
+  - rubocop-rspec
+  - rubocop-faker
+
+AllCops:
+  UseCache: false
+  TargetRubyVersion: <%= RUBY_VERSION[/\d+\.\d+/] %>
+  Exclude:
+    - bin/*
+    - db/schema.rb
+    - db/seeds.rb
+    - config/puma.rb
+    - config/environments/*
+
+Style/NumericPredicate:
+  Enabled: false
+
+Style/Lambda:
+  EnforcedStyle: literal
+
+Style/Documentation:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
I think it's better remove this. Highly likely Ruby 3+ will have string literals frozen by default. So it's better to be prepared :)